e2e/utils/test-utils.ts

```diff
@@ -1,9 +1,98 @@
-import {Page} from 'puppeteer';
+import {JSHandle, Page} from 'puppeteer';
 import Checkbox from 'app/element/checkbox';
 import RadioButton from 'app/element/radiobutton';
 import TextOptions from 'app/element/text-options';
 import Textarea from 'app/element/textarea';
 import Textbox from 'app/element/textbox';
+import GoogleLoginPage from 'app/page/google-login';
+import HomePage, {LABEL_ALIAS} from 'app/page/home-page';
+import Link from '../app/element/link';
+import {waitForText} from './wait-utils';
+
+export async function signIn(page: Page) {
+  const loginPage = new GoogleLoginPage(page);
+  await loginPage.login();
+  // this element exists in DOM after user has logged in
```

Comment:
Is there something more user-facing, and less likely to change, that we could use to detect a successful login?